molecule/default/verify.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Check whether packages are installed
+  hosts: all
+  become: yes
+
+  tasks:
+
+    - name: Verify utilities are available
+      command: >
+        {{ item }}
+        with_items:
+      with_items:
+        - git --version
+        - curl --version
+        - htop --version
+        - iotop --version
+        - make --version
+        - multitail -h
+        - ncdu -v
+        - nload --help
+        - pv --version
+        - tree --version
+        - unzip --help
+        - bc --version
+        - screen --version
```

Comment:
Screen also apparently punishes you for simply asking about its version. Try `screen -d -m true` instead (from https://superuser.com/a/454914/34981).